README.md

```diff
@@ -40,15 +44,33 @@ sudo pacman -S pandoc texlive-core
 #### OSX
 ```bash
 brew install pandoc
-mkdir $HOME/context
-cd $HOME/context
-rsync -av rsync://contextgarden.net/minimals/setup/first-setup.sh .
-sh ./first-setup.sh --modules=all --engine=luatex
+brew cask install mactex
```

Comment:
Ah.. I double checked this and `context` is not part of `mactex`. Will have to update this again to revert the above 5 lines.